cpp/include/cudf/lists/detail/gather.cuh

```diff
@@ -112,6 +107,11 @@ gather_data make_gather_data(cudf::lists_column_view const& source_column,
                       return src_offsets[index];
                     });
 
+  // now that we are doing using the gather_map, we can release the underlying prev_base_offsets.
```

Comment:
Just curious to know the argument for not using "we", as it's definitely something I'm guilty of doing a lot :)

Reply:
It is just something I learning about technical writing - is that you should try to avoid it when possible. It is usually quite easy:
```cpp
// now that we are doing using the gather_map, we can release the underlying 
```
to 
```cpp
// due to the use of gather_map, the underlying ... can be released
```
Use of pronouns is generally better for more informal writing.

Reply:
In my opinion comments are informal writing, and pronouns are fine. It is very difficult to use active voice without pronounce. Notice the passive voice in your second example. I think active voice is more important.

As for #1 above, I think maybe @nvdbaranec meant "now that we are done using"?